sdcm/cluster_aws.py

```diff
@@ -693,6 +693,14 @@ def traffic_control(self, tcconfig_params=None):
             tc_command = LOCAL_CMD_RUNNER.run("tcset eth1 {} --tc-command".format(tcconfig_params)).stdout
             self.remoter.run('sudo bash -cxe "%s"' % tc_command)
 
+    def install_traffic_control(self):
+        self.log.debug("Check that traffic_control is installed")
+        res = self.remoter.run("/sbin/tc -h", ignore_status=True)
+        if not res.ok:
+            self.log.debug("traffic control package is not installed. Installing...")
+            self.pkg_install("iproute-tc")
```

Comment:
Wow, first usage of this method since 2y after it's introduction :-)

There is no advantage over `self.remoter.run("yum install -y iproute-tc")` in the way you call it.